lib/testing_api_onlyoffice_com/test_instance/main_page/document_builder_api/document_builder_api_common/builder_page.rb

```diff
@@ -48,7 +48,11 @@ def open_integrating_document_builder
     end
 
     def menu_data
-      @menu_data ||= JSON.parse(File.read("#{__dir__}/document_entries.json"))
```

Comment:
Maybe replace usage of `menu_data` to usage of `parse_document_entries`?
Methods do almost same things except caching